cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/transaction/distributed/TransactionServiceTest.java

```diff
@@ -202,6 +210,12 @@ static TransactionService createTxService(String zkConnectionString, int txServi
                            new ZKClientModule(),
                            new DiscoveryRuntimeModule().getDistributedModules(),
                            new TransactionMetricsModule(),
+                           new AbstractModule() {
+                             @Override
+                             protected void configure() {
+                               bind(NamespaceQueryAdmin.class).to(SimpleNamespaceQueryAdmin.class);
```

Comment:
@bdmogal There are other places where we use Modules.override. I would suggest we can leave this as it is for now and revisit this again when we address CDAP-6534.